cpython-windows/build.py

```diff
@@ -1865,11 +1863,13 @@ def main() -> None:
         default="cpython-3.11",
         help="Python distribution to build",
     )
+    # TODO: Rename this to `--options` to match the Unix build script
+    optimizations = {"debug", "noopt", "pgo", "lto", "pgo+lto"}
     parser.add_argument(
-        "--profile",
-        choices={"noopt", "pgo"},
+        "--options",
+        choices=optimizations.union({f"freethreaded+{o}" for o in optimizations}),
         default="noopt",
-        help="How to compile Python",
+        help="Build options to apply when compiling Python",
     )
```

Comment:
Two things to follow up on here

- I renamed this, I should remove the stale comment.
- I added more optimization types here, that's wrong I should remove the other options.

Reply:
(Hesitant to do that in this pull request unless there are other changes since it takes so long to do builds)

Reply:
```diff
diff --git a/cpython-windows/build.py b/cpython-windows/build.py
index 3425a3b..92bca7d 100644
--- a/cpython-windows/build.py
+++ b/cpython-windows/build.py
@@ -1863,8 +1863,7 @@ def main() -> None:
         default="cpython-3.11",
         help="Python distribution to build",
     )
-    # TODO: Rename this to `--options` to match the Unix build script
-    optimizations = {"debug", "noopt", "pgo", "lto", "pgo+lto"}
+    optimizations = {"noopt", "pgo"}
     parser.add_argument(
         "--options",
         choices=optimizations.union({f"freethreaded+{o}" for o in optimizations}),
```